src/config/config.py

```diff
@@ -0,0 +1,109 @@
+"""Module to handle configuration application configuration parameters."""
+import configparser
+import logging
+from os.path import expanduser, isfile
+from pathlib import Path
+
+logger = logging.getLogger(__name__)
+config_file = "config.ini"
+
+
+def find_config_directory() -> str:
+    """Get configuration directory.
+
+    Returns
+    -------
+    str     :
+        Path represented as a string to a directory.
+        Ends with a trailing /
+    """
+    # TODO: Check if windows or linux, use enviroment variables
+    return expanduser("~") + "/.config/nina/"
+
+
+def get_config_file_path() -> str:
+    """Get configuration file path.
+
+    Returns
+    -------
+    str     :
+        Path represented as a string to the configuration file.
+    """
+    return find_config_directory() + config_file
+
+
+def get_default_config() -> configparser.ConfigParser:
+    """Get default settings stored in ConfigParser object.
+
+    Returns
+    -------
+    ConfigParser    :
+        ConfigParser object containing default values.
+    """
+    default_config = configparser.ConfigParser()
+    default_config["DEFAULT"]["hostname"] = "127.0.0.1"
+    default_config["DEFAULT"]["enable"] = "true"
+    default_config["GLOBAL"] = {}
+    default_config["GLOBAL"]["development"] = "false"
+    default_config["UI"] = {}
+    default_config["UI"]["port"] = "5000"
+    default_config["CORE"] = {}
+    default_config["CORE"]["port"] = "8000"
+    default_config["CORE"]["batch_size"] = "100"
```

Comment:
```suggestion
    default_config["CORE"]["batch_size"] = "100"
    default_config["CORE"]["database_path"] = ""
```
This should also maybe have it's own function to get default path depending on os.

Reply:
Should maybe put this in `.local/share/nina/` folder? And then something equivalent for windows.

Reply:
Seems reasonable

Reply:
`XDG_DATA_HOME`